app_dart/test/request_handlers/update_agent_health_test.dart

```diff
@@ -46,6 +49,8 @@ void main() {
       expect(agent.agentId, 'test');
       expect(response.agent.isHealthy, true);
       expect(agent.healthDetails, 'bar detail');
+      expect(int.parse(response.totalRows), 1);
```

Comment:
Since this test has access to FakeTabledataResourceApi, it could read the result from there instead of a response.

Reply:
updated.